### PR TITLE
Added postinstall script to build styled-components.

### LINKED ISSUE #2425: Tests fail when running `yarn test`
I'm trying to take a pass at fixing an issue at styled-components source code. Full disclosure, I'm new to the repo and I don't have a full understanding how things are structured here.
 
I've forked styled-components and ran `yarn` and then `yarn test`. I was following the instructions [here](https://github.com/styled-components/styled-components/blob/master/CONTRIBUTING.md#code-contributions).

I was hoping all tests would pass since I haven't changed anything. But 2 tests fail:

screen shot 2019-03-04 at 12 00 52 pm

Is that expected?

## System:
 - OS: macOS 10.14.3
 - CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
 - Memory: 2.19 GB / 16.00 GB
 - Shell: 5.7.1 - /usr/local/bin/zsh
## Binaries:
 - Node: 8.12.0 - /usr/local/bin/node
 - Yarn: 1.12.3 - /usr/local/bin/yarn
 - npm: 6.4.1 - /usr/local/bin/npm

### PR DESCRIPTION
**Pull request by alansouzati**
Closes #2425

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "size": "lerna run size --stream --parallel",
     "test": "run-s test:*",
     "prepublishOnly": "yarn clean && yarn && yarn build && yarn test && cp README.md packages/styled-components",
+    "postinstall": "yarn bs && yarn build",
```
Comment (probablyup) [inline package.json]: Does this run on normal install as a dependency as well? We only want it to run for local installations.

Comment (alansouzati) [reply package.json]: good catch. yeah I believe this is wrong, let me take a look

Comment (alansouzati) [reply package.json]: I believe I've fixed this by moving to `prepare`

**Comment by alansouzati** [comment]
FYI I've also changed travis config to remove `yarn build` since `yarn` runs build now.

**Pull request merged**